src/Harness/Startup/DependencyInjection.cs

```diff
@@ -0,0 +1,40 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.S3;
+using Harness.Contracts;
+using Harness.Repositories;
+using Harness.Services;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Harness.Startup;
+
+public static class DependencyInjection
+{
+    public static IServiceCollection RegisterServices(this IServiceCollection services, IWebHostEnvironment env,
+        IConfiguration configuration)
+    {
+        // Add services to the container.
+        services.AddControllers();
+        services.AddEndpointsApiExplorer();
+        services.AddSwaggerGen();
+
+        // check if in development
+        if (env.IsDevelopment())
```

Comment:
The harness is a dev only tool, so I would remove this check.
You want the harness to always use the DI configured below not fallback to the handler defaults in another env.